Design.md

```diff
@@ -3,21 +3,13 @@
 Let's briefly elucidate the design decisions behind the creation of the **concrete.biopython** library.
 
 ## Objective: Alignment with Biopython
-The primary goal is to align the `concrete.biopython` library with the principles adopted by the `concrete.fhe` Python library. In the case of `concrete.fhe`, the approach is to ensure compatibility and interoperability with the `numpy` library. This is achieved by offering analogous functions in the `concrete.fhe.array` class as those in the `numpy.array` class, allowing seamless transition between unencrypted and encrypted states using `numpy` functions with `concrete.fhe.array` objects.
+The primary goal is to align the `concrete.biopython` library with the principles adopted by the `concrete.fhe` Python library. In the case of `concrete.fhe`, the approach is to ensure compatibility and interoperability with the `numpy` library. This is achieved by offering a compatibility layer between the `concrete.fhe.array` class and `numpy.array` class, allowing seamless transition between unencrypted and encrypted states using `numpy` functions with `concrete.fhe.array` objects.
```

Comment:
I think you missed the point. `concrete.fhe.array` is **not** a `class`.

Reply:
ok, I corrected the phrasing